pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "wheel", "setuptools_dso>=2.1", "epicscorelibs>=7.0.7.99.1.1a3"]
+requires = ["setuptools", "wheel", "setuptools_dso>=2.1", "epicscorelibs>=7.0.7.99.1.2a1"]
```

Comment:
Eww, seems a bit arbitrary.  This pinning looks all a bit uncomfortable to be, I suppose there's a purpose for it?